pytest/lib/mocknet.py

```diff
@@ -951,21 +952,24 @@ def create_upgrade_schedule(rpc_nodes, validator_nodes, progressive_upgrade,
                 else:
                     prev_stake = prev_stake + STAKE_STEP
                     staked = prev_stake * ONE_NEAR
-                stakes.append((staked, node))
+                stakes.append((staked, node.instance_name))
                 print(f'{node_account_name(node.instance_name)} {staked}')
 
         else:
-            staked = MIN_STAKE
+            for i, node in enumerate(validator_nodes):
+                stakes.append((MIN_STAKE, node.instance_name))
```

Comment:
Unnecessary `enumerate` and unused `i`. This also could be done with a generator:

    stakes.extend((MIN_STAKE, node.instance_name) for node in validator_nodes)